Dockerfile.delve

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine3.18 AS build
+FROM golang:1.21-alpine3.19 AS build
```

Comment:
We can't update to alpine 3.19 yet, see https://github.com/grafana/rollout-operator/pull/97. It's deceiving because integration tests are passing here, but it's because `integration/mock-service/Dockerfile` hasn't been updated in this PR.

Reply:
The above has been resolved now, but the main Dockerfile now compiles in bookworm. I don't think that difference matters here besides consistency.